torchvision/prototype/transforms/functional/_geometry.py

```diff
@@ -1434,8 +1431,8 @@ def ten_crop_video(video: torch.Tensor, size: List[int], vertical_flip: bool = F
 
 
 def ten_crop(
-    inpt: features.ImageOrVideoTypeJIT, size: List[int], vertical_flip: bool = False
-) -> List[features.ImageOrVideoTypeJIT]:
+    inpt: Union[features.ImageTypeJIT, features.VideoTypeJIT], size: List[int], vertical_flip: bool = False
+) -> Union[List[features.ImageTypeJIT], List[features.VideoTypeJIT]]:
```

Comment:
I haven't seen that before. So this works, but it doesn't for tuples? :exploding_head:

Reply:
Yes. This is possibly because they do various mitigations for Tuples as we've seen previously.